integreat_cms/api/v3/locations.py

```diff
@@ -115,7 +116,13 @@ def locations(request, region_slug, language_slug):
     # Throw a 404 error when the language does not exist or is disabled
     region.get_language_or_404(language_slug, only_active=True)
     result = []
-    pois = region.pois.prefetch_public_translations().filter(archived=False)
+    pois = (
+        region.pois.prefetch_public_translations()
+        .filter(archived=False)
+        .select_related("category")
+    )
+    # TODO category translations need to be prefetched
```

Comment:
```suggestion
        .prefetch_related("category__translations__language")
    )
```
> So I decided to add (copy) the prefetch methods to poi_category.py, but the result is not saved, "prefetched_translations" is not available, so always a db call is made. I do not really know how i should call these methods in locations.py. Can you help please?

For the other content models, the prefetching is implemented as custom queryset methods:

https://github.com/digitalfabrik/integreat-cms/blob/28277a074118b7f6f6307ea6b1a87901a4de9977/integreat_cms/cms/models/abstract_content_model.py#L39-L48

However, since POI categories are a lot simpler, we can just rely on the inbuilt Django queryset method [`prefetch_related()`](https://docs.djangoproject.com/en/4.1/ref/models/querysets/#prefetch-related).